.travis.yml

```diff
@@ -23,12 +21,12 @@ matrix:
   fast_finish: true
   include:
     - os: linux
-      node_js: "6"
+      node_js: "8"
       env: CXX=g++-4.8 TEST_SUITE=coveralls
     - os: linux
-      node_js: "6"
+      node_js: "8"
       env: CXX=g++-4.8 TEST_SUITE=lint
     - os: linux
-      node_js: "6"
-      env: CXX=g++-4.8 TEST_SUITE=test:browser
```

Comment:
So the browser tests and not run anymore? Are they useful?

Reply:
That would be separate construction site. They were broken and I decided that it's better to have the main tests running/being checked and having failure indicated there.

The browser tests are useful to see if the library can be converted to be used in the browser I suppose.